src/main/kotlin/com/workos/mfa/MfaApi.kt

```diff
@@ -247,6 +251,76 @@ class MfaApi(private val workos: WorkOS) {
     return workos.post("/auth/factors/verify", VerifyFactorResponse::class.java, config)
   }
 
+  /**
+   * Parameters for the [verifyChallenge] method.
+   */
+  @JsonInclude(Include.NON_NULL)
+  class VerifyChallengeOptions constructor(
+    @JsonProperty("authentication_challenge_id")
```

Comment:
I think we still need to add it here?
```suggestion
    @JsonIgnore
```

Reply:
Once we update that we should be good to merge @pliao28